src/patternfly/components/Table/table-grid.scss

```diff
@@ -112,16 +118,40 @@
   // Table toggle icons
   --pf-c-table__toggle__icon--Transition: .2s ease-in 0s;
   --pf-c-table__toggle--m-expanded__icon--Transform: rotate(180deg);
-
-  // End responsive vars
-  // ==================================================================
 }
 
 // Check table cell
 @include pf-mobile-layout {
+  --pf-c-table--cell--PaddingTop: var(--pf-c-table--m-grid--cell--PaddingTop);
+  --pf-c-table--cell--PaddingRight: var(--pf-c-table--m-grid--cell--PaddingRight);
+  --pf-c-table--cell--PaddingBottom: var(--pf-c-table--m-grid--cell--PaddingBottom);
+  --pf-c-table--cell--PaddingLeft: var(--pf-c-table--m-grid--cell--PaddingLeft);
+
   display: grid;
   border: none;
 
+  // reset cell modifications
+  &.pf-c-table tr > * {
+    width: auto;
+    min-width: 0;
+    max-width: none;
+    overflow: visible;
+    text-overflow: clip;
```

Comment:
why does the default text-overflow for mobile need to be set to `clip`

Reply:
Just resetting to its default value